gcp_variant_transforms/libs/vcf_header_parser.py

```diff
@@ -55,6 +55,22 @@ def get_vcf_headers(input_file):
   return HeaderFields(vcf_reader.infos, vcf_reader.formats)
 
 
+def get_header_lines(input_file):
```

Comment:
nit: s/get_header_lines/get_metadata_header_lines
and clarify that it only returns lines starting with `##` and not `#`